scripts/deployer.ts

```diff
@@ -0,0 +1,197 @@
+import { ethers } from 'hardhat';
+import { HardhatRuntimeEnvironment } from 'hardhat/types';
+import { DeployFunction } from 'hardhat-deploy/types';
+import 'hardhat-deploy-ethers';
+import hre from 'hardhat';
+import * as fs from 'fs';
+import { ContractReceipt, ContractTransaction } from 'ethers';
+import mainnetData from '../mainnet_deployed.json';
+import testnetData from '../testnet_deployed.json';
+import dummyData from '../example_testnet_deployed.json';
+import { getCurrentTimestamp } from 'hardhat/internal/hardhat-network/provider/utils/getCurrentTimestamp';
+
+let deployedData = dummyData;
```

Comment:
The `mainnetData`, `testnetDat` and `dummyData` share the same structure right? I would propose creating Interface with this structure which you would declare here and then you should not need the `@ts-ignore` bellow when you reassign it with mainnet/testnet data...

Please double check what I wrote, most probably there is some mistake somewhere as I am not writing it in IDE 😅

```suggestion

interface DeployedContract {
      abi: Array<any>, // Improve these types
      bytecode: string,
      address: string,
      block: number,
      url: string
} 
interface DeployedData {
	contracts: {
		postage: DeployedContract,
		redistribution: DeployedContract,
		staking: DeployedContract,
		priceOracle: DeployedContract,
		factory: DeployedContract
	}
}

let deployedData: DeployedData = dummyData;
```